core/arch/x86/decode_table.c

```diff
@@ -6785,6 +6785,10 @@ const instr_info_t evex_W_extensions[][2] = {
     {OP_vpxord, 0x660fef10, "vpxord", Ve, xx, KEw, He, We, mrm|evex, x, END_LIST},
     {OP_vpxorq, 0x660fef50, "vpxorq", Ve, xx, KEb, He, We, mrm|evex, x, END_LIST},
   },
+  {    /* evex_W_ext 45 */
+    {OP_vpmulld,  0x66384018, "vpmulld",   Ve, xx, KEw,He,We, mrm|evex|reqp, x, END_LIST},
+    {OP_vpmullq,  0x66384058, "vpmullq",   Ve, xx, KEb,He,We, mrm|evex|reqp, x, END_LIST},
```

Comment:
When I do Intel syntax the comma is in the wrong place:
```
  0x000056299b12af0e  62 e2 f5 47 40 41 37 vpmullq zmm16{, k7}, zmm17, [rcx+0x37]
```

Reply:
(Trying to get confidence in the changes but I seem to hit a different problem with each attempt to spot-check.)

Reply:
The instruction PR's haven't aimed at fixing all problems parallel to AVX-512. Intel's syntax may very well not work correctly it seems. This one here is (lack of support in-) 7be31a6bac9980fbc9e.

Reply:
It seems best to do disasm checks on each set of new instructions as they're added, to have more than just internal self-consistency.  I thought this was already being done on at least a subset of each new set, in some manner.  If these other disasm things are fixed first it should be feasible to feed this test output to other decoders and have an automated workflow to do first-order checks with minimal effort on each set.

Reply:
(Not regression tests which are more work to set up: just a quick script workflow vs capstone or llvm or objdump which will catch most things, with those binutils tests coming in later for regression.)

Reply:
> It seems best to do disasm checks on each set of new instructions as they're added

Which _has_ been done.

> I thought this was already being done on at least a subset of each new set, in some manner.

It has.

> If these other disasm things are fixed first it should be feasible to feed this test output to other decoders and have an automated workflow to do first-order checks with minimal effort on each set.

Maybe talk offline what exactly you're missing.

Reply:
I filed #3638 to cover my proposal as a regression test.